### ISSUE TITLE
after running the node, the deployment contract reports an error

### ISSUE THREAD

**Issue by robertcc0410** [b2network/b2-node#48]
1. ./init.sh
2. deploy contract

The error message is as follows：
ProviderError: rpc error: code = Unknown desc = invalid opcode: PUSH0

**Comment by robertcc0410** [comment]
Error when deploying contract and running
```ProviderError: rpc error: code = Unknown desc = invalid opcode: PUSH0
    at HttpProvider.request (b2network/b2-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at HardhatEthersProvider.estimateGas (b2network/b2-contracts/node_modules/@nomicfoundation/hardhat-ethers/src/internal/hardhat-ethers-provider.ts:237:27)
    at b2network/b2-contracts/node_modules/@nomicfoundation/hardhat-ethers/src/signers.ts:235:35
    at async Promise.all (index 0)```

**Comment by tony-armstrong** [comment]
see [fix: solidity version downgrade #1 · b2network/b2-zkevm-contracts@9033fc8](https://github.com/b2network/b2-zkevm-contracts/commit/9033fc8b026b30d56ac2f93666996fd5b526a32a)
